README.md

```diff
@@ -72,9 +72,24 @@ By default, view nodes are strongly typed. The type of a container includes the
 
 The type erasure of View nodes is not an easy trick, as the trait has two associated types and the `rebuild` method takes the previous view as a `&Self` typed parameter. Nonetheless, it is possible. (As far as I know, Olivier Faure was the first to demonstrate this technique, in [Panoramix], but I'm happy to be further enlightened)
 
+## Prerequisites
+
+### Linux
+
+In order to run xilem on linux systems, the development library [xkbcommon] needs to be installed.
+
+On Fedora this can be installed using:
+
+```sh
+sudo dnf install libxkbcommon-x11-devel.x86_64
+# i686 is a less common architecure, but if you have it, use libxkbcommon-x11-devel.i686 instead
+```
+
+
```

Comment:
forgot to add debian dep, not sure if i686 applies here as well?

```suggestion
Debian:

```sh
sudo apt-get install libxkbcommon-x11-dev
```
```